app/controllers/users_controller.rb

```diff
@@ -1,13 +1,17 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show, :edit, :update]
 
+  load_and_authorize_resource
+
   def index
     @users = User.all
   end
 
   def show
     @cards = @user.latest_cards
     @liked = @user.latest_likes
+    @total_likes = @user.total_likes
+    @total_cards = @user.total_cards
```

Comment:
well i thought it would be better if all of these guys are in one place. and as for views Im passing local variables to 'statistics' partials which is good. 
Also this is not an ideal thing in my opinion. it makes two count queries. I would love it if it does not do that. we will need to look into some ways to cache this information.